.github/ISSUE_TEMPLATE/unreproducible_package.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Unreproducible package
+about: A package that does not produce a bit-by-bit reproducible result each time it is built
+title: ''
+labels: '0.kind: enhancement', '6.topic: reproducible builds'
+assignees: ''
+
+---
+
+Building this package twice does not produce the bit-by-bit identical result each time, making it harder to detect CI breaches. You can read more about this at https://reproducible-builds.org/ .
+
+Fixing bit-by-bit reproducibility also has additional advantages, such as avoiding hard-to-reproduce bugs, making content-addressed storage more effective and reducing rebuilds in such systems.
+
+### Steps To Reproduce
+
+    nix-build '<nixpkgs>' -A ... --check --keep-failed
+
```

Comment:
````suggestion

```
nix-build '<nixpkgs>' -A ... --check --keep-failed
```

```` 
easier to paste the logs and follows the other templates